guides/common/modules/proc_configuring-compliance-policy-deployment-methods.adoc

```diff
@@ -1,31 +1,32 @@
 [id="configuring-compliance-policy-deployment-methods_{context}"]
 = Configuring Compliance Policy Deployment Methods
 
-Use the following procedure to configure {Project} for the method that you have selected to deploy compliance policies.
-
-.Procedure
-* For the Ansible deployment method:
-** Import the `theforeman.foreman_scap_client` Ansible role.
+Use one the following procedures to configure {Project} for the method that you have selected to deploy compliance policies.
 
+.Procedure: Ansible deployment
```

Comment:
Maybe replace ": " with " for "; drop ".Procedure "; or reword to "Deploying Compliance Policies using Ansible"